build/ci/Makefile

```diff
@@ -77,6 +87,13 @@ ci-release: vault-public-key vault-docker-creds
     	cloud-on-k8s-ci-release \
     	bash -c "make -C operators ci-release"
 
+yaml-upload: vault-aws-creds
+	@ AWS_ACCESS_KEY_ID=$(shell cat $(VAULT_AWS_ACCESS_KEY_FILE)) \
+		AWS_SECRET_ACCESS_KEY=$(shell cat $(VAULT_AWS_SECRET_KEY_FILE)) \
+		aws s3 cp \
+		$(ROOT_DIR)/operators/config/all-in-one.yaml \
+		s3://download.elasticsearch.org/downloads/eck/$(TAG_NAME)/all-in-one.yaml
```

Comment:
What about the other deployment manifests (single namespace, mixed)? Can be done in another PR I guess but maybe raise another issue if it is not happening in this one.

Reply:
Created an issue for that https://github.com/elastic/cloud-on-k8s/issues/912